migration-tools/modules/migration-tools-commons-tests/src/main/java/org/apache/ignite/migrationtools/tests/containers/Ignite3ClusterContainer.java

```diff
@@ -183,4 +181,15 @@ public IgniteClient.Builder clientBuilder() {
     public IgniteClient buildClient() {
         return clientBuilder().build();
     }
+
+    /**
+     * Retrieves the ignite 3 docker image name which should be used in the tests.
+     *
+     * @return The docker image name.
+     */
+    public static String dockerImageName() {
+        String imageName = System.getProperty("ignite3.docker.image");
+        assert imageName != null : "ignite3.docker.image property must be defined";
```

Comment:
Is this supposed to be seen by the users? I think an exception might be better than an assertion.

Reply:
Ah, I've noticed `src/main` but not on the module name. Does not matter then.